webapp/components/needs_team.jsx

```diff
@@ -103,6 +106,11 @@ export default class NeedsTeam extends React.Component {
         });
 
         Utils.applyTheme(this.state.theme);
+
+        if (UserAgent.isIosSafari()) {
```

Comment:
I put it under NeedsTeam because the pages underneath of it (channel view, permalink view, and to a lesser extent, the tutorial) are the ones where we don't want the body of the page scrolling since it interferes with the content scrolling. I tried applying it higher up, but it prevented scrolling on the login and team selection pages where we do want that to be possible. Having it here prevents it from negatively affecting pages where it doesn't need to be.